### ISSUE TITLE
High level description of stacker.news codebase

### ISSUE THREAD

**Issue by SouthKoreaLN** [stackernews/stacker.news#1089]
**Is your feature request related to a problem? Please describe.**

Just wondering if some other _new_ contributors to Stacker.news would like to have a high-level introduction of how everything fits together in the stacker.news codebase. I am truly new to all of this, not just stacker.news, so it could be that this would not benefit anyone else but me. In such case, it is of course not worth the trouble to draft something. And I am for sure just learning by doing, constantly `grep`ping to find the places where things are defined and implemented, called and stored, etc.

**Describe the solution you'd like**

Add an `md` file in the docs folder that describes the structure, the high-level logic linking the different parts, the philosophy behind certain choices (stateless, etc), some preferred practices, etc. 
This could then be loaded of course in the FOSS homepage (see #1058).

**Describe alternatives you've considered**

The alternative is nearly as good, i.e. just learning by doing.

**Additional context**

Please don't waste time on this if I am the only one that would benefit from this.

**Comment by adlai** [comment]
Thank you for searching and finding the issue @Soxasora ; @huumn why did you remove the `documentation` label?

**Comment by adlai** [comment]
> **Describe alternatives you've considered**
> 
> The alternative is nearly as good, i.e. just learning by doing.

no, that ruins the commons in favor of trite shite that sounds good on the vibe.

dead good documents that painstakingly outline domain-specific problems and solutions are never replaceable by tweet-length good advice that is hopelessly generic.
